packages/ui-components/src/components/radio-button-group/radio-button-group.tsx

```diff
@@ -0,0 +1,32 @@
+import { Component, h, Prop, Host, Listen, Event, EventEmitter } from '@stencil/core';
+import { IRadioButton } from '../radio-button/radio-button.types';
+
+@Component({
+	tag: 'kv-radio-button-group',
+	styleUrl: 'radio-button-group.scss',
+	shadow: true
+})
+export class KvRadioButtonGroup {
+	/** (required) List of radio buttons */
+	@Prop({ reflect: true }) buttons: { [key: string]: IRadioButton } = {};
+
+	/** When the radio button selection changes, emit the requested tab's key */
+	@Event() radioButtonChange: EventEmitter<string>;
```

Comment:
This event should emit all radio buttons selected, right? Not only that changed.

Reply:
This event is emitted every time that a button is checked and return the id of button checked

Reply:
I got it, I was just saying that since it's a group, the state change should be regarding all buttons instead of only one.